db/crud.go

```diff
@@ -2220,14 +2220,18 @@ func (db *DatabaseCollectionWithUser) getChannelsAndAccess(ctx context.Context,
 
 	if db.channelMapper() != nil {
 		// Call the ChannelMapper:
-		startTime := time.Now()
 		db.dbStats().Database().SyncFunctionCount.Add(1)
+		db.dbStats().CollectionStat(db.ScopeName(), db.Name()).SyncFunctionCount.Add(1)
```

Comment:
What do you think about storing a reference to the CollectionStats on DatabaseCollectionWithUser when it's initialized, to avoid the map lookup every time we want to write a stat?  (that's especially true if the CollectionStat call ends up needing a mutex)